src/components/feedbacks/FeedbackItem.vue

```diff
@@ -33,4 +29,11 @@ import ArrowUp from "../icons/ArrowUp.vue";
 import IconComments from "../icons/IconComments.vue";
 import BaseBox from "../basicComponents/BaseBox.vue";
 import BaseButton from "../basicComponents/BaseButton.vue";
+
+defineProps({
+  feedback: {
+    type: Object,
+    default: null,
```

Comment:
If null then you should think about hiding this component

Reply:
I added this as a suggestion - I will ask you on Monday about that cause I am not sure if I understand well what do you mean